lib/assets/javascripts/cartodb/table/menu_modules/filters/templates/filters.jst.ejs

```diff
@@ -17,7 +17,7 @@
   <div class="scrollpane">
     <ul class="filters"></ul>
     <div class="add_filter">
-      <a href="#" class="add">+</a>
+      <a href="#" class="add">+ Add Filter</a>
```

Comment:
Let's add some left margin to this, since its currently smashed up against the left edge of the parent viewport. Maybe 10px?